File: 프로그래머스/1/12922. 수박수박수박수박수박수？/수박수박수박수박수박수？.cs
public class Solution {
    public string solution(int n) {
        string answer = "";
        string[] arr = {"수", "박"};
        
        for(int i = 0; i < n; i++)
        {
            if(i % 2 == 0) answer += arr[0];
            else answer += arr[1];
        }
        return answer;
    }
}

File: 프로그래머스/1/12922. 수박수박수박수박수박수？/수박수박수박수박수박수？.cpp
#include <string>
#include <vector>

using namespace std;

string solution(int n) {
    if(n == 1) return "수";
    int num = n / 2;
    string answer = "";
    
    for(int i = 0; i < num; i++)
    {
        answer += "수박";
    }
    return n % 2 == 0 ? answer : answer += "수";
}